chore(docs): document list escape hatches (#1342)
hashicorp/terraform-cdk: website/docs/cdktf/concepts/providers-and-resources.html.md

}
```

> You can access elements in arrays by a dot notation: `resource.addOverride("configurations.0.https", true)`.

Reviewer: ```suggestion
Use a dot notation to access elements in arrays: `resource.addOverride("configurations.0.https", true)`.
```
